.php_cs

```diff
@@ -0,0 +1,80 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(__DIR__);
+
+$fixers = array(
```

Comment:
I guess its a starting point, having the config in place means this can eventually be formalized, and the `php-cs-fixer` checks can be added to travis once that is done. 

There are a lot of formatting changes in this PR - I didn't even manage to review them all.

I've just been looking over the PSRs, and Fixers defined for them and this config still includes a lot of symfony cs fixers. I might revert the previous commit and re-run php-cs-fixer with only the PSR based fixers.